### ISSUE TITLE
Test we use index lookup as expected for top-level collections

### ISSUE THREAD

**Issue by Totktonada** [tarantool/graphql.0#72]
Check in metainfo that we find needed indexes as expected.

* [ ] filter fields correspond to full set of key parts
* [ ] filter fields cover full set of key parts
* [ ] fillter fields correspond to partial set of key parts
* [ ] fillter fields cover partial set of key parts

Blocked by #71.

**Comment by Totktonada** [comment]
I'm going to archive the repository. I'll proceed as follows:

* Mark all open pull requests with the `archived` label and close.
* Mark all open issues with `archived` and `wontfix` labels and close.
* Archive the repository.

Consider the following projects:

* https://github.com/tarantool/graphql
* https://github.com/tarantool/graphqlapi
* https://github.com/tarantool/graphqlide
